src/components/KweetsList/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactElement } from 'react';
+import styled from 'styled-components';
+import { Kweet as KweetModel } from '../../model/kweet';
+import { Kweet } from '../Kweet';
```

Comment:
this line should go above `import { Kweet as KweetModel } from '../../model/kweet';`